lib/common.bash

```diff
@@ -158,6 +158,20 @@ extract_kata_env(){
 	NETMON_PATH="/usr/libexec/kata-containers/kata-netmon"
 }
 
+# Checks that containerd-shim-kata-v2 processes are not running
+check_ctr_shim_processes() {
```

Comment:
nitpick:
We could still call `kata-runtime kata-env` without any issue and it'd bring some info about the system.
But I totally understand the reason for not calling it.

Reply:
That ends up doing a lot of parsing, etc, to get the same info I think we can get from which.

Long term that may be more robust, but I probably won't have cycles to add that.